scripts/package.ps1

```diff
@@ -53,10 +79,20 @@ $System.AutomaticManagedPagefile = $true
 $System.Put()
 
 Write-BoxstarterMessage "Setting up winrm"
-Set-NetFirewallRule -Name WINRM-HTTP-In-TCP-PUBLIC -RemoteAddress Any
-Enable-WSManCredSSP -Force -Role Server
+if ($OS.Version -eq "6.1.7601") {
+    C:\Windows\System32\netsh.exe advfirewall firewall add rule name="WinRM-HTTP" dir=in localport=5985 protocol=TCP action=allow
+    } else {
+    Set-NetFirewallRule -Name WINRM-HTTP-In-TCP-PUBLIC -RemoteAddress Any
+}
+
+if ($OS.Version -eq "6.1.7601") {
+    Enable-PSRemoting -Force -SkipNetworkProfileCheck
```

Comment:
I had to switch the CredSSP command order here, or errors about not being able to connect to WinRM were generated